.github/workflows/scripts/generate-weekly-report.js

```diff
@@ -151,6 +155,11 @@ async function getIssuesData({octokit, context}) {
       count: 0,
       data: []
     },
+    issuesCodeOwnerNeeded: {
+      title: "New issues needing code owners",
```

Comment:
```suggestion
      title: "Issues and PRs that need code owner review",
```
The purpose of this label is essentially saying that the given issue or PR is essentially blocked until a code owner shares their thoughts. Whether it's an approval on a PR or "final say" on design decision in the component.

I'm open to different wording here, whatever is most clear.